devtools/git/pre-commit

```diff
@@ -85,7 +100,7 @@ else
   FMTFAILED=0
   for file in $(git diff --name-only --cached);
   do
-    if [ -f "$file" ] && [ ${file: -3} == ".rs" ]; then
+    if [ -f "$file" ] && [ ${file: -3} == ".rs" ] && [ $(contains "${IGNORE[@]}" "$file") == "n" ] ; then
```

Comment:
I recommend using `case` here to filter out files that should be ignored.

Reply:
updated